scripts/base/frameworks/files/main.zeek

```diff
@@ -61,7 +61,7 @@ export {
 		depth: count &default=0 &log;
 
 		## A set of analysis types done during the file analysis.
-		analyzers: set[string] &default=string_set() &log;
+		analyzers: set[string] &ordered &log;
```

Comment:
(Wouldn't it  be nice if `&ordered` just sorted them alphabetically right away ...)

Reply:
We have a ticket with `&sorted`.